openmrs-19/src/main/java/org/motechproject/openmrs19/service/impl/OpenMRSConfigServiceImpl.java

```diff
@@ -53,16 +63,17 @@ public void postConstruct() {
     public void deleteAllConfigs() {
         configs.getConfigs().clear();
         configs.setDefaultConfigName(null);
-        updateConfigs();
     }
 
     @Override
     @Transactional
     public void saveAllConfigs(Configs configs) {
```

Comment:
I didn't notice this earlier, but we are shadowing the configs field with this method parameter. Can we rename the method parameter to avoid it?